utils/cargo-tfhe-lints/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use std::process::{exit, Command};
+
+fn get_supported_rustc_version() -> &'static str {
+    const TOOLCHAIN_FILE: &str = include_str!("../rust-toolchain-link");
+
+    TOOLCHAIN_FILE
+        .lines()
+        .find(|line| line.starts_with("channel ") || line.starts_with("channel="))
```

Comment:
starts_with("channel") should be enough ?

Reply:
fixed